### ISSUE TITLE
Dark Mode Quick Toggle

### ISSUE THREAD

**Issue by dhishkyaon** [JavaCafe01/PdfViewer#141]
Add a quick toggle for Dark Mode because pictures doesn't show right in the Dark Mode and user has to go under the settings to turn it on/off.
It would be great if user has capability to change the mode directly when reading the pages.

**Comment by Pomegranate123** [comment]
Yes, either this or having some setting to ignore images when inverting colors for dark mode.
